api/node/index.ts

```diff
@@ -292,9 +294,10 @@ export class CompileError extends Error {
 /**
  * Loads the given slint file and returns a constructor to create an instance of the exported component.
  */
-export function loadFile(path: string) : Object {
+export function loadFile(filePath: string) : Object {
+    let absoluteFilePath = path.resolve(filePath);
     let compiler = new napi.ComponentCompiler;
-    let definition = compiler.buildFromPath(path);
+    let definition = compiler.buildFromPath(absoluteFilePath);
```

Comment:
I agree that this is a good workaround for now. Please add a comment here that explains why this is necessary. I'll look into fixing it.

Reply:
done